options/locale/locale_en-US.ini

```diff
@@ -2474,6 +2474,7 @@ branch.create_branch_operation = Create branch
 branch.new_branch = Create new branch
 branch.new_branch_from = Create new branch from "%s"
 branch.renamed = Branch %s was renamed to %s.
+branch.copy_branch_name = Copy Branch Name
```

Comment:
The text could be `Copy branch name`, to match other texts like `Create new branch`

Reply:
See suggestion below